tests/test_helper/parallelize.bash

```diff
@@ -0,0 +1,69 @@
+# This file contains set of helper functions to run processes or functions
```

Comment:
Having unused code in a repository adds unnecessary maintenance burden, should we perhaps postpone adding it until we have a concrete use for it?

Reply:
`parallelize.bash` file removed